sci-libs/vtk/vtk-7.0.0.ebuild

```diff
@@ -0,0 +1,345 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+WEBAPP_OPTIONAL=yes
+WEBAPP_MANUAL_SLOT=yes
+
+inherit flag-o-matic java-pkg-opt-2 python-single-r1 qmake-utils versionator toolchain-funcs cmake-utils virtualx webapp
+
+# Short package version
+SPV="$(get_version_component_range 1-2)"
+
+DESCRIPTION="The Visualization Toolkit"
+HOMEPAGE="http://www.vtk.org/"
+SRC_URI="
+	http://www.vtk.org/files/release/${SPV}/VTK-${PV}.tar.gz
+	doc? ( http://www.vtk.org/files/release/${SPV}/vtkDocHtml-${PV}.tar.gz )
+	test? (
+		http://www.vtk.org/files/release/${SPV}/VTKData-${PV}.tar.gz
+		http://www.vtk.org/files/release/${SPV}/VTKLargeData-${PV}.tar.gz
+		)
+	"
+
+LICENSE="BSD LGPL-2"
+KEYWORDS="~amd64 ~arm ~x86 ~amd64-linux ~x86-linux"
+SLOT="0"
+IUSE="
+	all-modules aqua boost doc examples imaging ffmpeg gdal java json kaapi mpi
+	mysql odbc offscreen postgres python qt5 rendering tbb test theora tk tcl
+	video_cards_nvidia views web xdmf2 R +X"
+
+REQUIRED_USE="
+	all-modules? ( python xdmf2 boost )
+	java? ( qt5 )
+	python? ( ${PYTHON_REQUIRED_USE} )
+	tcl? ( rendering )
+	test? ( python )
+	tk? ( tcl )
+	web? ( python )
+	^^ ( X aqua offscreen )
+	"
+
+RDEPEND="
+	boost? ( dev-libs/boost:=[mpi?] )
+	dev-libs/expat
+	dev-libs/jsoncpp:=
+	dev-libs/libxml2:2
+	examples? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		sci-libs/vtkdata
+	)
+	ffmpeg? ( virtual/ffmpeg )
+	gdal? ( sci-libs/gdal )
+	java? ( >=virtual/jre-1.5:* )
```

Comment:
There's a couple of problems there. Because of `DEPEND="${RDEPEND}"` below, you've got jre and jdk in `DEPEND`. This can cause problems but in this particular case, switching to a JRE would break anyway because of the way VTK links against AWT. In short, this should be changed to `java? ( >=virtual/jdk-1.7:* )` (not 1.5)  and the jdk line below should be removed.